tailscale/assets/monitors/physical_traffic_received.json

```diff
@@ -0,0 +1,56 @@
+{
+	"version": 2,
+	"created_at": "2023-11-07",
+	"last_updated_at": "2023-11-07",
+	"title": "High Physical Traffic Received by Destination",
+	"description": "Use this monitor to set an alert if the rate of change for physical traffic from a specific source is abnormally high.",
+	"definition": {
+		"name": "Unusually High Physical Traffic To Destination: {{[@physicalTraffic.dst].name}} ",
+		"type": "log alert",
+		"query": "formula(\"derivative(query)\").last(\"15m\") > 100000",
+		"message": "Use this monitor to set an alert if the rate of change for physical traffic from a specific source is abnormally high. The correct threshold value to use will vary by environment. To determine the correct value for your organization, use the average rate of change for  the physical traffic widget in the out-of-the-box dashboard. A good starting value is 5x the average value you've seen over the last 4 hours. The value can be adjusted as needed.",
+		"tags": [],
+		"options": {
+			"thresholds": {
+				"critical": null
+			},
```

Comment:
It looks like the CI is complaining because of this thresholds. Did you manually modify the json or is there something wrong with the validation?

Reply:
@FlorentClarret Nope, no manual updates were made to the JSON. We did not want to specify a threshold here due to the fact that can be wildly different per customer use case, so I think the validation should allow for that.